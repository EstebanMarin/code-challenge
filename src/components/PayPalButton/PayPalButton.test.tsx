import React from 'react'
import { render } from '@testing-library/react'
import PayPalButtonButton from './index'

test('Renders correctly the header', () => {
  const test = render(<PayPalButtonButton />)
  expect(test).toMatchSnapshot()
})
