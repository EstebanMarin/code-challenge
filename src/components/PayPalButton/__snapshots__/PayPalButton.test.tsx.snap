// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly the header 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <button
        class="sc-AxjAm fVXeNo"
      >
        <div
          class="sc-AxirZ dqdiuJ"
        >
          <img
            class="sc-AxiKw bMyFfp"
            src="icon-paypal-logo.svg"
          />
          <img
            class="sc-AxiKw bMyFfp"
            src="Arrow.png"
          />
        </div>
      </button>
    </div>
  </body>,
  "container": <div>
    <button
      class="sc-AxjAm fVXeNo"
    >
      <div
        class="sc-AxirZ dqdiuJ"
      >
        <img
          class="sc-AxiKw bMyFfp"
          src="icon-paypal-logo.svg"
        />
        <img
          class="sc-AxiKw bMyFfp"
          src="Arrow.png"
        />
      </div>
    </button>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
